src/main/kotlin/rx/lang/kotlin/observables.kt

```diff
@@ -107,3 +87,12 @@ fun <T, R> List<Observable<T>>.zip(zipFunction: (args: List<T>) -> R): Observabl
  * Returns an Observable that emits the items emitted by the source Observable, converted to the specified type.
  */
 inline fun <reified R : Any> Observable<*>.cast(): Observable<R> = cast(R::class.java)
```

Comment:
agreed on you with that but 
```
val strings = 
            Observable.just("Alpha", 1, "Beta").ofType<String>()
```
Makes sence, it's up to develpoer to write a good code